mobify/progressive-web-scaffold
# Global Component: Mini-Cart

Exploratory work on the global minicart component.

 **JIRA**: [WEB-922](https://mobify.atlassian.net/browse/WEB-922)

## Changes
- [x] Styles and markup
- ~~Engineering~~ (will be separate PR)

## How to test-drive this PR
- Run `npm run dev`
- Preview Merlin's Potions and open the Mini-Cart
- Confirm that the mini-cart is styled like the mockups
